DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/zero_copy_data_transfer/src/CMakeLists.txt

```diff
@@ -21,7 +21,15 @@ endif()
 # to ensure the BSP has the required support. Allow the user to define USM_HOST_ALLOCATIONS_ENABLED
 # to override this check (e.g., cmake .. -DUSM_HOST_ALLOCATIONS_ENABLED=1)
 if(NOT FPGA_BOARD MATCHES ".usm.*" AND NOT DEFINED USM_HOST_ALLOCATIONS_ENABLED)
-    message(FATAL_ERROR "This tutorial requires a BSP that has USM host allocations enabled")
+    message(FATAL_ERROR "This tutorial requires a BSP that has USM host allocations enabled.")
+endif()
+
+if (DEFINED USM_HOST_ALLOCATIONS_ENABLED)
```

Comment:
These are meant to be bools, right?  Why not just if (USM_HOST_ALLOCATIONS_ENABLED) ?  That way you don't have to print the value in order to 'use' this variable and avoid the warning.  I think we do this in other samples.

Reply:
Done